src/components/Header/Brand/Brand.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from '../../../styles/brand.module.css'
+export const Brand = () => {
+    return (
+        <div className={styles.brand}>
+             <span><i className="fas fa-cog fa-spin fa-3x  fa-spin"></i></span>
```

Comment:
Hi Maddy,
I haven't gone through the entire code will update more once I do, there are some changes from UI/UX perspective which I would recommend (however there are just suggestions if you feel very strong about it you can keep as it is):
1. Please stop the spinning of the log after few seconds or just add it on hover, it's a bit distracting